app/test_main.py

```diff
@@ -1,3 +1,25 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,expected_result",
+    [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 27, [2, 2]),
+        (28, 28, [3, 2]),
+        (28, 29, [3, 3]),
+        (100, 100, [21, 17])
```

Comment:
The test cases provided cover a range of typical scenarios, but according to the checklist, you should also test edge cases such as negative numbers or extremely large values. Consider adding these tests to ensure the function handles all possible inputs correctly.